src/5e-SRD-Magic-Items.json

```diff
@@ -244,6 +244,90 @@
       "If the bag is pierced or torn, it is destroyed, and anything contained within it is transported to a random location on the Astral Plane."
     ],
     "url": "/api/magic-items/bag-of-devouring"
+  },
+   {
+    "index": "bag-of-holding",
+    "name": "Bag of Holding",
+    "equipment_category": {
+      "name": "Wondrous Item",
+      "url": "/api/equipment-categories/wondrous-item"
+    },
+    "desc": [
+      "Wondrous	item,	uncommon",
```

Comment:
Looks like there is some weird spacing here.

Reply:
woah wtf